great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -49,9 +52,46 @@ def __init__(
             runtime_environment=runtime_environment,
         )
 
+    def list_expectation_suite_names(self) -> List[str]:
+        """
+        Lists the available expectation suite names. If in ge_cloud_mode, a list of
+        GE Cloud ids is returned instead.
+        """
+        if self.ge_cloud_mode:
+            return [
+                suite_key.ge_cloud_id for suite_key in self.list_expectation_suites()
+            ]
+
+    @property
+    def ge_cloud_config(self) -> Optional[GeCloudConfig]:
+        return self._ge_cloud_config
+
+    @property
+    def ge_cloud_mode(self) -> bool:
+        return self._ge_cloud_mode
+
     def _init_variables(self) -> CloudDataContextVariables:
         raise NotImplementedError
 
+    def _construct_data_context_id(self) -> str:
+        """
+        Choose the id of the currently-configured expectations store, if available and a persistent store.
+        If not, it should choose the id stored in DataContextConfig.
+        Returns:
+            UUID to use as the data_context_id
+        """
+
+        # if in ge_cloud_mode, use ge_cloud_organization_id
+        return self.ge_cloud_config.organization_id
+
+    def _save_project_config_to_disk(self) -> None:
+        """Save the current project to disk."""
+        if self.ge_cloud_mode:
+            logger.debug(
+                "ge_cloud_mode detected - skipping DataContext._save_project_config"
```

Comment:
I believe these should both be log messages (to prevent a valid usecase (EphemeralDataContext and CloudDataContext) from having to throw an error every time.